modules/tinymce/src/plugins/table/main/ts/core/TableSections.ts

```diff
@@ -81,9 +81,11 @@ const switchRowSection = (dom: DOMUtils, rowElm: HTMLElement, newSectionName: st
   }
 };
 
-const switchCellType = (dom: DOMUtils, cells: ArrayLike<HTMLTableCellElement>, newCellType: string) => {
-  Arr.each(cells, (c) => Util.getNodeName(c) !== newCellType ? dom.rename(c, newCellType): c);
-};
+const switchCellType = (dom: DOMUtils, cells: ArrayLike<HTMLTableCellElement>, newCellType: string, scope: 'col' | null) =>
+  Arr.each(cells, (c) => {
+    const newCell = Util.getNodeName(c) !== newCellType ? dom.rename(c, newCellType): c; // creates new element
```

Comment:
The `colon` needs a space?

Reply:
oh, huh. how come eslint didn't pick up on that... thanks!